packages/toolpad-core/src/PageContainer/PageHeader.tsx

```diff
@@ -94,7 +94,7 @@ function PageHeader(props: PageHeaderProps) {
                   {getItemTitle(item)}
                 </Link>
               ) : (
-                <Typography key={item.path} color="text.primary">
+                <Typography key={item.title} color="text.primary">
```

Comment:
Let's use the `map` index here instead. We don't necessarily expect `title` to be uniqye

Reply:
Great thinking! Done!